Source/PrebidNativeNative/PrebidNativeAd.swift

```diff
@@ -0,0 +1,317 @@
+/*   Copyright 2019-2020 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+import UIKit
+
+@objcMembers public class PrebidNativeAd: NSObject, CacheExpiryDelegate {
```

Comment:
Why do we have the folder "PrebidNativeNative"?
Why do we have the prefix in every class? "PrebidNativeAdDelegate", "PrebidNativeAd", "PrebidNativeAdEventDelegate"?
We have never used the naming for all other classes

Reply:
"PrebidNativeNative" was used to group those classes which only used for prebid native implementation. I have updated them now.  
Also removed all the references of prefix "prebid".